src/Castle.Facilities.Logging/LoggerResolver.cs

```diff
@@ -22,11 +22,6 @@ namespace Castle.Facilities.Logging
 	using Castle.MicroKernel;
 	using Castle.MicroKernel.Context;
 
-	/// <summary>
-	///   Custom resolver used by Windsor. It gives
-	///   us some contextual information that we use to set up a logging
-	///   before satisfying the dependency
-	/// </summary>
```

Comment:
Why remove the comment?

Reply:
Dont really like XML comments, they are sometimes not relevant or describe the mundanely obvious. It is leads to a yo-yo effect in the code file with regards to scrolling. Apart from giving you nice descriptions in the VS tooling I don't see much value in having them. Can put these back if you want, I don't feel strongly that they should be removed. Just makes reading code easier.